api/go.mod

```diff
@@ -8,8 +8,8 @@ require (
 	github.com/openstack-k8s-operators/lib-common/modules/storage v0.1.0
 	github.com/openstack-k8s-operators/openstack-ansibleee-operator/api v0.1.0
 	github.com/openstack-k8s-operators/openstack-baremetal-operator/api v0.1.0
-	k8s.io/api v0.26.7
-	k8s.io/apimachinery v0.26.7
+	k8s.io/api v0.27.1
+	k8s.io/apimachinery v0.27.1
```

Comment:
Hmm, I thought we were avoiding 0.27.1 for now. Need to double check the rest of the projects to see if we wanted this bump.